test/spec/Feature/Query/DeleteSpec.hs

```diff
@@ -90,7 +243,7 @@ spec =
 
       it "succeeds deleting the row when return=representation and selecting only the privileged columns" $
         request methodDelete "/app_users?id=eq.1&select=id,email" [("Prefer", "return=representation")]
-          [json| { "password": "passxyz" } |]
+          mempty
```

Comment:
What happens when there are filters applied and a request body too?

Reply:
Yeah, I did not add a test for that. It treats the request body as the main source for PKs and adds the query string as filters (works like an intersection of PKs between the body and the query string). The WHERE clause for the delete looks something like this:

```sql 
...
WHERE app_users.id IN <request_body>.id AND id = <query_string.id> AND <other_filters>
```

Before changing this test, there were no ids in the body (only a password column) so it didn't delete anything.

Reply:
I see. Hmm, for this case I recall the bulk update did not consider the body PKs as filters. Would it be possible to do this for DELETE as well? I think we should have the same behavior for both bulk update and delete.

Reply:
> Now, to keep consistency, as soon as any filter (not only PKs) exist in the query string when deleting, then the body should be ignored (there's no data to be used for updating here), correct?

Yes, exactly. That would be consistent with PATCH.